### ISSUE TITLE
Explorer can't Show two directories whose names have the same letters but differ in capitalization.

### ISSUE THREAD

**Issue by imjiangjun** [microsoft/vscode#86362]
Issue Type: <b>Bug</b>

I copy some directories to the workfolder on remote machine. I can see them using `ls` in terminal. But they don't appear in the explorer even I refresh it many times. If I try to make a new directory with the same name in vscode explorer, It will warn that the directory already exists. Note that I use remote development (SSH).

VS Code version: Code 1.40.2 (f359dd69833dd8800b54d458f6d37ab7c78df520, 2019-11-25T14:54:45.096Z)
OS version: Windows_NT x64 10.0.18362
Fetching remote diagnostics for 'ssh-remote+7b22686f73744e616d65223a223137322e31382e33322e3238222c2275736572223a226a69616e676a756e227d' timed out.

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 3192)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.84GB (9.87GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

Fetching remote diagnostics for 'ssh-remote+7b22686f73744e616d65223a223137322e31382e33322e3238222c2275736572223a226a69616e676a756e227d' timed out.
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
bracket-pair-colorizer|Coe|1.0.61
vscode-language-pack-zh-hans|MS-|1.40.2
remote-containers|ms-|0.83.1
remote-ssh|ms-|0.47.2
remote-ssh-edit|ms-|0.47.2
remote-wsl|ms-|0.40.3
vscode-remote-extensionpack|ms-|0.17.0
path-intellisense|chr|1.4.2
vscode-markdownlint|Dav|0.32.0
githistory|don|0.4.10
gitlens|eam|10.2.0
todo-tree|Gru|0.0.162
git-graph|mhu|1.19.1
vscode-language-pack-zh-hans|MS-|1.40.2
python|ms-|2019.11.50794
cpptools|ms-|0.26.2
vscode-yaml|red|0.6.1
markdown-preview-enhanced|shd|0.5.0
vscodeintellicode|Vis|1.2.0

</details>

**Comment by imjiangjun** [comment]
Issue Type: <b>Bug</b>

I copy some directories to the workfolder on remote machine. I can see them using `ls` in terminal. But they don't appear in the explorer even I refresh it many times. If I try to make a new directory with the same name in vscode explorer, It will warn that the directory already exists. Note that I use remote development (SSH).

VS Code version: Code 1.40.2 (f359dd69833dd8800b54d458f6d37ab7c78df520, 2019-11-25T14:54:45.096Z)
OS version: Windows_NT x64 10.0.18362
Fetching remote diagnostics for 'ssh-remote+7b22686f73744e616d65223a223137322e31382e33322e3238222c2275736572223a226a69616e676a756e227d' timed out.

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 3192)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.84GB (9.87GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

Fetching remote diagnostics for 'ssh-remote+7b22686f73744e616d65223a223137322e31382e33322e3238222c2275736572223a226a69616e676a756e227d' timed out.
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
bracket-pair-colorizer|Coe|1.0.61
vscode-language-pack-zh-hans|MS-|1.40.2
remote-containers|ms-|0.83.1
remote-ssh|ms-|0.47.2
remote-ssh-edit|ms-|0.47.2
remote-wsl|ms-|0.40.3
vscode-remote-extensionpack|ms-|0.17.0
path-intellisense|chr|1.4.2
vscode-markdownlint|Dav|0.32.0
githistory|don|0.4.10
gitlens|eam|10.2.0
todo-tree|Gru|0.0.162
git-graph|mhu|1.19.1
vscode-language-pack-zh-hans|MS-|1.40.2
python|ms-|2019.11.50794
cpptools|ms-|0.26.2
vscode-yaml|red|0.6.1
markdown-preview-enhanced|shd|0.5.0
vscodeintellicode|Vis|1.2.0


</details>

**Comment by IllusionMH** [comment]
/needsMoreInfo

@imjiangjun Are you sure you don't have his folder in **Files: Exclude** (`files.exclude`) setting for both User and Workspace/Folders configurations?

---

_Off-topic: @chrmarti maybe it make sense to add bot command that will allow to remove initial labels assigned by bot? Currently many issues has `integrated-terminal` assigned when it is not related_

**Comment by imjiangjun** [comment]
@IllusionMH I don't have this setting. This is my `.vscode/settings.json`:
```
{
    "python.pythonPath": "/mnt/storage/jiangjun/program/anaconda3/envs/py36/bin/python",
    "python.formatting.autopep8Args": [
        "--max-line-length=120",
        "--aggressive"
    ]
}
```
And it is not in `.gitignore` neither.
By the way, where to set `file:exclude`? I don't find it.

I tried to **rename** the diretory (e.g. as `WWH`). Then it **appeared** in vscode explorer. After that, I **renamed it back** as `EF_B5`, and it **disappeared again**! It should be a very ordinary directory. That's really strange.

**Comment by IllusionMH** [comment]
You can open Settings <kbd>Ctrl</kbd>+<kbd>,</kbd> and should see 3 tabs under search field: User, Workspace, and Remote SSH. Paste `files.exclude` into search field and check if there is anything else in list except defaults:

```json
	"files.exclude": {
		"**/.git": true,
		"**/.svn": true,
		"**/.hg": true,
		"**/CVS": true,
		"**/.DS_Store": true
	},
```

**Comment by imjiangjun** [comment]
@IllusionMH Thank you. I checked the three settings as you said. They are all kept default.

**Comment by IllusionMH** [comment]
Could you provide examples of folders that are hidden?

/cc @isidorn for visibility

**Comment by imjiangjun** [comment]
I finally find what happened.
There are two directories `ef_b5` and `EF_B5` under the same path. They have the same name but one is capital and the other one is lowercase.
If I change the name of any one of them to differ it from the other one, both will appear.
That means the explorer can't show two directories whose names have the same letters but differ in capitalization.

**Comment by isidorn** [comment]
I believe there already might an issue for this.
If you want you can search for duplicates and upvote that issue.
It might be in the vscode-remote repo. Sorry for the vague pointers :)

**Comment by IllusionMH** [comment]
Root case (and this can be closed as duplicate of) #12448

Exact duplicate https://github.com/microsoft/vscode-remote-release/issues/973
